pkg/shell/manifest.json.in

```diff
@@ -13,7 +13,8 @@
         "pl-pl": "polski",
         "pt-br": "português (Brasil)",
         "uk-ua": "Українська",
-        "zh-cn": "中文（中国）"
+        "zh-cn": "中文（中国）",
+        "cs-cz": "čeština"
```

Comment:
Could this please stay sorted?

Reply:
Definitely! I thought it was ordered, but seems not. I will fix it.